robot/EDA/tests/browser/pe/program_enrollments.robot

```diff
@@ -1,28 +1,28 @@
 *** Settings ***
 
-Resource        robot/HEDAP/resources/HEDA.robot
-Library         robot/HEDAP/resources/HEDA.py
+Resource        robot/EDA/resources/EDA.robot
+Library         robot/EDA/resources/EDA.py
 Suite Setup     Open Test Browser
 Suite Teardown  Delete Records and Close Browser
 
 *** Test Cases ***
 
-Verify HEDA Settings
-    Go To Heda Settings
+Verify EDA Settings
+    Go To Eda Settings
 
-    Wait for Locator                            heda_settings.affiliations_tab
```

Comment:
@dashattuck  Not urgent (can look into it tomorrow - it's getting late :)) - but there is a robot test failure and I think it's potentially related to what was changed here?